app/commands/decidim/vocdoni/admin/create_election.rb

```diff
@@ -58,10 +56,8 @@ def create_election!
         def election_type_attributes
           {
             election_type: {
-              auto_start: form.auto_start,
               interruptible: form.interruptible,
               dynamic_census: form.dynamic_census,
-              secret_until_the_end: form.secret_until_the_end,
               anonymous: form.anonymous
```

Comment:
this attributes are still in the step of creating he election? I think all these steps go in the last step